src/main/resources/assets/mmorpg/lang/ru_ru.json

```
@@ -0,0 +1,2718 @@
+	"mmorpg.stat.magic_shield_per_sec": "Магический Щит за секунду",
+	"mmorpg.stat.magic_shield_regen": "Восстановление Магического Щита",
+	"mmorpg.stat.mana": "Мана",
+	"mmorpg.stat.mana_cost": "Затрата маны",
+	"mmorpg.stat.mana_leech_cap": "Максимальная скорость высасывания Маны",
+	"mmorpg.stat.mana_on_hit_hit": "Мана за удар",
+	"mmorpg.stat.mana_on_kill": "Мана за убийство",
+	"mmorpg.stat.mana_per_10_intelligence": "§a[VAL1] §9☯ Мана§7 на каждые 10 §b★ Интеллекта§7",
+	"mmorpg.stat.mana_per_sec": "Мана в секунду",
+	"mmorpg.stat.mana_regen": "Восстановление Маны",
+	"mmorpg.stat.mana_shield": "От поглощаемого урона Маной",
+	"mmorpg.stat.manasteal": "Кража Маны",
+	"mmorpg.stat.max_all_resist": "Максимальное сопротивление всем",
+	"mmorpg.stat.max_chaos_resist": "Максимальное сопротивление Хаосу",
+	"mmorpg.stat.max_elemental_resist": "Максимальное сопротивление стихиям",
+	"mmorpg.stat.max_fire_resist": "Максимальное сопротивление Огню",
+	"mmorpg.stat.max_lightning_resist": "Максимальное сопротивление Молнии",
+	"mmorpg.stat.max_physical_resist": "Максимальное сопротивление Физическому урону",
+	"mmorpg.stat.max_total_summons": "Максимальное количество призванных существ",
+	"mmorpg.stat.max_water_resist": "Максимальное сопротивление Холоду",
+	"mmorpg.stat.mining_core_drop_bonus": "Основные выпадения",
+	"mmorpg.stat.mining_double_drop": "Шанс на удвоенный выпад",
+	"mmorpg.stat.mining_exp_bonus": "Бонус опыта в профессии",
+	"mmorpg.stat.mining_greater_drop_bonus": "Эпические выпадения",
+	"mmorpg.stat.mining_lesser_drop_bonus": "Обычные выпадения",
+	"mmorpg.stat.mining_medium_drop_bonus": "Редкие выпадения",
+	"mmorpg.stat.mining_misc_drop_bonus": "Прочие выпадения",
+	"mmorpg.stat.mining_triple_drop": "Шанс на тройной выпад",
+	"mmorpg.stat.minion_explode_cdr": "Снижение перезарядки взрыва миньона",
+	"mmorpg.stat.minion_explode_spell_dmg": "Урон от взрыва миньона",
+	"mmorpg.stat.more_food_stats": "§7Вы получаете §a[VAL1]§7% больше характеристик от баффов еды.",
+	"mmorpg.stat.more_threat_on_take_dmg": "§7Вы генерируете на §a[VAL1]§7% больше угрозы при получении урона.",
+	"mmorpg.stat.move_speed": "Скорость передвижения",
+	"mmorpg.stat.movement_cdr": "Снижение перезарядки заклинаний передвижения",
+	"mmorpg.stat.movement_spell_dmg": "Урон от заклинаний передвижения",
+	"mmorpg.stat.negative_eff_dur_u_cast": "Длительность негативного эффекта",
+	"mmorpg.stat.negative_eff_duration": "§7§a[VAL1]§7% длительности негативных эффектов на вас.",
+	"mmorpg.stat.non_crit_damage": "Урон без критических атак",
+	"mmorpg.stat.offensive_eff_dur_u_cast": "Длительность агрессивного эффекта",
+	"mmorpg.stat.offensive_eff_duration": "§7§a[VAL1]§7% длительности агрессивных эффектов на вас.",
+	"mmorpg.stat.out_of_combat_regen": "Восстановление вне боя",
+	"mmorpg.stat.phys_to_all": "Физический урон во все виды урона",
+	"mmorpg.stat.phys_to_chaos": "Физический урон в урон Хаоса",	
+	"mmorpg.stat.phys_to_elemental": "Физический урон в урон стихиям",
```

```suggestion
	"mmorpg.stat.phys_to_elemental": "Физический урон в урон Стихиями",
```
С большой буквы пишу лишь потому, что в остальных подобных статах тип урона с большой

---

Исправлено